app/models/manageiq/providers/hawkular/middleware_manager/middleware_domain.rb

```diff
@@ -0,0 +1,4 @@
+module ManageIQ::Providers
```

Comment:
but we don't have it for all models, right? so, do we need it for domains/server groups? (the jms PR doesn't have it iirc)